### PR TITLE
fix(ui5-dialog): add text with instructions for resizing or dragging of dialog fo…

### LINKED ISSUE #6659: [SF][A11y][Dialog] Resizable dialog, header getting focus its not announcing help text to resize the dialog.
### Describe the bug

Describe the bug:
Resizable dialog announcement is missing when focus is moved to the header.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/story/modals-popovers-dialog--default&args=resizable:!true

### Reproduction steps

Go to [Dialog Document Page](https://sap.github.io/ui5-webcomponents-react/?path=/story/modals-popovers-dialog--default&args=resizable:!true)
Click on 'Open Dialog' button
Inspect speech history of NVDA screen reader
See the announcement of Dialog, when focus is moved to header.
Screenshot 2023-03-07 at 12 09 25 PM

### Expected Behaviour

When Dialog header gets the focus it should announce "Use Shift+Arrow keys to resize". 

Example is taken from SAPUI5 dialog, 

https://openui5.hana.ondemand.com/#/entity/sap.m.Dialog/sample/sap.m.sample.Dialog
Screenshot 2023-03-07 at 1 12 49 PM

### Screenshots or Videos

_No response_

### UI5 Web Components for React Version

1.8.1

### UI5 Web Components Version

1.10.4

### Browser

Chrome

### Operating System

MacOS 13.2.1 (22D68)

### Additional Context

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by LidiyaGeorgieva**
Fixes #6659

### CODE REVIEW

**Review by TeodorTaushanov — CHANGES_REQUESTED** [review]
Add aria-roledescription = "Interactive Header"
Add also text for dragging the dialog:
aria-describedby = "Use Arrow keys to move, Shift+Arrow keys to resize"

**Review by TeodorTaushanov — CHANGES_REQUESTED** [review]
How the "describedby text" is not read by screen readers. 

Create a new "div" inside the "header" with role="group" and put all the logic there. Check the Openui5 Dialog control.

Check if the css styles should be adapted.

```
<header>
			<div
				class="ui5-popup-header-root"
				id="ui5-popup-header"
				role="group"
				aria-describedby={{effectiveAriaDescribedBy}}
				aria-roledescription={{ariaRoleDescriptionHeaderText}}
				tabindex="{{_headerTabIndex}}"
```

**Comment by LidiyaGeorgieva** [comment]
> 

It works on my machine :)
image
Tested on: chrome 113, Jaws 2022, Win 11

File: packages/main/src/Dialog.hbs [hunk 1]
```diff
@@ -2,22 +2,39 @@
...
+				{{/if}}
+				{{#if header.length }}
+					<slot name="header"></slot>
+				{{else}}
+					<h1 id="ui5-popup-header-text" class="ui5-popup-header-text">{{headerText}}</h1>
+				{{/if}}
+
+				{{#if resizable}}
+					{{#if draggable}}
+					<span id="{{_id}}-descr" class="ui5-hidden-text">{{ariaDescribedByHeaderTextDraggableAndResizable}}</span>
```
Comment (TeodorTaushanov) [inline packages/main/src/Dialog.hbs:28]: Add aria-hidden="true" to the "describedby" text.

**Review by TeodorTaushanov — CHANGES_REQUESTED** [review]

**Review by TeodorTaushanov — APPROVED** [review]

**Pull request merged**